Tests/MaintenanceTests/MaintenanceTests.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  MaintenanceTests.swift
+//
+//
+//  Created by Emad Ghorbania on 18/11/2022.
+//
+
+import XCTest
+@testable import Maintenance
+import Firebase
+final class MaintenanceTests: XCTestCase {
```

Comment:
Should we also have a test case for both `fullscreen` and `banner` are false?

Reply:
On those tests when one of test one of them the other one is false, how would you suggest me to test the false?
like false false or what?

Reply:
Something like:
```
Maintenance.isThereAnyMaintenance(
            fullscreen: false,
            banner: false,
            fullscreenDetail: Maintenance.Detail(title: "", body: ""),
            bannerDetail: Maintenance.Detail(title: "", body: ""))
```
?